client/pd_service_discovery.go

```diff
@@ -518,6 +563,46 @@ func (c *pdServiceDiscovery) updateServiceModeLoop() {
 		}
 	}
 }
+func (c *pdServiceDiscovery) memberHealthCheckLoop() {
+	defer c.wg.Done()
+
+	memberCheckLoopCtx, memberCheckLoopCancel := context.WithCancel(c.ctx)
+	defer memberCheckLoopCancel()
+
+	ticker := time.NewTicker(MemberHealthCheckInterval)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-c.ctx.Done():
+			return
+		case <-ticker.C:
+			c.checkLeaderHealth(memberCheckLoopCtx)
+			c.checkFollowerHealth(memberCheckLoopCtx)
+		}
+	}
+}
+
+func (c *pdServiceDiscovery) checkLeaderHealth(ctx context.Context) {
+	ctx, cancel := context.WithTimeout(ctx, c.option.timeout)
+	defer cancel()
+	leader := c.getLeaderServiceClient()
+	leader.checkNetworkAvailable(ctx)
+}
+
+func (c *pdServiceDiscovery) checkFollowerHealth(ctx context.Context) {
+	c.followers.Range(func(key, value any) bool {
+		// To ensure that the leader's healthy check is not delayed, shorten the duration.
+		ctx, cancel := context.WithTimeout(ctx, MemberHealthCheckInterval/3)
```

Comment:
why not `c.option.timeout`?

Reply:
`MemberHealthCheckInterval` is shorter than `defaultPDTimeout`